lib/screens/v2/candidate_generator/dup.ex

```diff
@@ -0,0 +1,79 @@
+defmodule Screens.V2.CandidateGenerator.Dup do
+  @moduledoc false
+
+  alias Screens.Config.Screen
+  alias Screens.Config.V2.Dup
+  alias Screens.Config.V2.Header.CurrentStopId
+  alias Screens.Stops.Stop
+  alias Screens.V2.CandidateGenerator
+  alias Screens.V2.Template.Builder
+  alias Screens.V2.WidgetInstance.{NormalHeader, Placeholder}
+
+  @behaviour CandidateGenerator
+
+  @impl CandidateGenerator
+  def screen_template do
+    {:screen,
+     %{
+       screen_normal: [
+         {:rotation_zero,
+          %{
+            body_normal_zero: [
+              :header_zero,
+              :main_content_primary_zero,
+              :inline_alert_zero
+            ],
+            screen_takeover_zero: [:full_screen_zero]
+          }},
+         {:rotation_one,
+          %{
+            body_normal_one: [
+              :header_one,
+              :main_content_primary_one
+            ],
+            screen_takeover_one: [:full_screen_one]
+          }},
+         {:rotation_two,
+          %{
+            body_normal_two: [
+              :header_two,
+              :main_content_secondary_two,
+              :inline_alert_two
+            ],
+            screen_takeover_two: [:full_screen_two]
+          }}
+       ]
+     }}
```

Comment:
Need a few tweaks to this, based on some discussion with Paul this morning.

The main change is this:
- make inline (or "partial") alerts optional, and give them their own dedicated slot rather than drawing directly over the main content slot

![IMG_1990](https://user-images.githubusercontent.com/63608771/213541378-c0b17b36-0421-4ce6-ba0a-d3799a19e431.jpg)

The rest are just suggested name changes to more closely align with the naming scheme of other existing templates.

![IMG_1989](https://user-images.githubusercontent.com/63608771/213530520-0ecbfb92-6c86-421e-9fed-093839cd27e1.jpg)

```suggestion
    {:screen,
     %{
       screen_normal: [
         {:rotation_zero,
          %{
            rotation_normal_zero: [
              :header_zero,
              {:body_zero,
               %{
                 body_normal_zero: [
                   :main_content_zero
                 ],
                 body_split_zero: [
                   :main_content_reduced_zero,
                   :bottom_pane_zero
                 ]
               }}
            ],
            rotation_takeover_zero: [:full_rotation_zero]
          }},
         {:rotation_one,
          %{
            rotation_normal_one: [
              :header_one,
              :main_content_one
            ],
            rotation_takeover_one: [:full_rotation_one]
          }},
         {:rotation_two,
          %{
            rotation_normal_two: [
              :header_two,
              {:body_two,
               %{
                 body_normal_two: [
                   :main_content_two
                 ],
                 body_split_two: [
                   :main_content_reduced_two,
                   :bottom_pane_two
                 ]
               }}
            ],
            rotation_takeover_two: [:full_rotation_two]
          }}
       ]
     }}
```

Reply:
(And, unfortunately, corresponding changes to the frontend layout components as well. Sorry!!)

Reply:
This makes sense. Made all the necessary changes to the frontend too.